README.md

```diff
@@ -1,8 +1,21 @@
 # Stedi EDI Bootstrap
 
-This repo contains an end-to-end configuration for building a full X12 EDI system using Stedi products. This implementation demonstrates one possible way to interact with Stedi's Products and APIs to achieve a typical EDI workload; your implementation may include some or all of these products depending on your particular systems and requirements.
+This repository contains an end-to-end configuration for building a full X12 EDI system using Stedi products. This implementation demonstrates one way to build an integration for the common read and write EDI use case. Your solution may differ, depending on your systems and requirements. We encourage you to [contact us](mailto:support@stedi.com) as you get started with Stedi, so we can help you design a custom integration.
 
-# Prerequisites & Deployment
+
+## Read and Write Workflow 
+
+The bootstrap workflow uses the following process to handle incoming EDI:
+
+1. An example trading partner uses [Stedi SFTP](https://www.stedi.com/products/sftp) to send EDI documents to a [Stedi bucket](https://www.stedi.com/products/buckets).
```

Comment:
A trading partner _could_ use SFTP (and in practice that's probably the common workflow), but in the instructions / testing section we have the user just upload the file using the Buckets UI, it might be better to indicate that here (could also add that uploads via SFTP will trigger this process as well)

Reply:
mm okay that's a good point. Let me adjust!